Tests/Unit/NetworkServer/LnsProtocolMessageProcessorTests.cs

```diff
@@ -284,29 +283,19 @@ internal async Task InternalHandleDataAsync_ShouldNotThrow_OnNonHandledMessageTy
                                                                        CancellationToken.None);
         }
 
-        private static Rxpk GetExpectedRxpk()
+        private static RadioMetadata GetExpectedRadioMetadata()
         {
             var radioMetadataUpInfo = new RadioMetadataUpInfo(0, 68116944405337035, 0, -53, (float)8.25);
```

Comment:
nit: Prefer type specifier over cast:

```suggestion
            var radioMetadataUpInfo = new RadioMetadataUpInfo(0, 68116944405337035, 0, -53, 8.25f);
```